guides/third_party_cdns.md

```diff
@@ -0,0 +1,25 @@
+# Third Party CDNs
+
+Moovweb XDN is designed and built to be the component of your site that your users will directly connect to from their devices. Such components are colloquially known as "edge components". But sometimes you may prefer, usually not due to technical reasons, to run XDN behind a third-party CDN. XDN fully supports this use case but it's important to call out some common pitfals with this kind of network topology.
```

Comment:
```suggestion
Moovweb XDN is designed and built to be the component of your site to which your users will directly connect to from their devices. Such components are colloquially known as "edge components". But sometimes you may prefer, usually not due to technical reasons, to run the XDN behind a third-party CDN due to a preexisting contract. XDN fully supports this use case but it's important to call out some common pitfals with this kind of network topology.
```

Reply:
If we add "due to a preexisting contract" we could then remove "usually not due to technical reasons"?

Reply:
Yes, I think so.

Reply:
I updated the suggestion